int32 axisID
int32 value
---
int32 res